latex/doc/latex.tex

```diff
@@ -53,7 +53,7 @@
 
 \dedication{\normalsize \textbf{Note:} Please note that this document has been created on
 \today. If you are using a devel version from GIT, please compile and check
-\texttt{doc/geanylatex.tex} from sources. Please check Page \pageref
```

Comment:
I woul'd rather keep the name of old version ins ChangeLog as they are.

Reply:
Maybe I am misunderstanding something but in that line it is only the name of the latex documentation. Not the name printed inside the log. Should I only change that line or a different one?

Reply:
At this very point the plugin was called GeanyLATeX so I'm voting for not changing that part of documentation, which is pretty much the ChangeLog.

Reply:
Oh. My fault. 
I pointed to the wrong line. Sorry.

Reply:
Ähemm..ok. Can you please point me to the right line? Or just send an PR.

Reply:
Completely my failure. Sorry. Nevermind.